test/grouping.jl

```diff
@@ -759,11 +776,15 @@ end
                 f(d -> (xyz=sum(d.c),), gd)
             @test f(col => (x -> sum(x)) => :xyz, gd) ==
                 f(d -> (xyz=sum(d.c),), gd)
-            @test f(col => (x -> (sum(x),)) => :xyz, gd) ==
-                f(d -> (xyz=(sum(d.c),),), gd)
-            @test_throws ArgumentError f(col => (x -> (z=sum(x),),) => :xyz, gd)
-            @test_throws ArgumentError f(col => (x -> DataFrame(z=sum(x),),) => :xyz, gd)
-            @test_throws ArgumentError f(col => (x -> (z=x,),) => :xyz, gd)
+            @test f(col => (x -> sum(x)) => :xyz, gd) ==
```

Comment:
This test is now identical to the one above. Why change this?

Reply:
OK - fixed. I thought that it was too much to test named tuple of tuples, but I have reverted it.